fractal-terrain-generation/dist/servePage.js

```diff
@@ -15,14 +17,21 @@ exports.handler = async (body, ctx) => {
     resourcePath = '/index.html';
   }
 
-  const webpage = await fs.readFile(`${prefix}${resourcePath}`);
+  let webPage;
```

Comment:
Since this code is mostly shared with `react-serverless`, perhaps also rename to `webResource`.

Also if this is to be a recommended code for using in a Binaris function perhaps makes senses moving it into a unified place.